pkg/tsdb/cloudwatch/models/cloudwatch_query.go

```diff
@@ -213,25 +214,9 @@ const timeSeriesQuery = "timeSeriesQuery"
 var validMetricDataID = regexp.MustCompile(`^[a-z][a-zA-Z0-9_]*$`)
 
 type metricsDataQuery struct {
-	Dimensions        map[string]interface{} `json:"dimensions"`
-	Expression        string                 `json:"expression"`
-	Label             *string                `json:"label"`
-	Id                string                 `json:"id"`
-	MatchExact        *bool                  `json:"matchExact"`
-	MetricEditorMode  *MetricEditorMode      `json:"metricEditorMode"`
-	MetricName        string                 `json:"metricName"`
-	MetricQueryType   MetricQueryType        `json:"metricQueryType"`
-	Namespace         string                 `json:"namespace"`
-	Period            string                 `json:"period"`
-	Region            string                 `json:"region"`
-	SqlExpression     string                 `json:"sqlExpression"`
-	Statistic         *string                `json:"statistic"`
-	Statistics        []*string              `json:"statistics"`
-	TimezoneUTCOffset string                 `json:"timezoneUTCOffset"`
-	QueryType         string                 `json:"type"`
-	Hide              *bool                  `json:"hide"`
-	Alias             string                 `json:"alias"`
-	AccountId         *string                `json:"accountId"`
+	dataquery.CloudWatchMetricsQuery
+	Type              string `json:"type"`
```

Comment:
still not understanding where Type is coming from
It's getting just set in the backend from QueryType? (sometimes?)